### DISCUSSION TITLE
Can I migrate my paid account to another PC?

### DISCUSSION THREAD

**Discussion by makkkkkraid** [aclap-dev/video-downloadhelper#1263] [License & Purchase requests]
### Your question

I am thinking of buying a premium license, but my current PC is old and I am planning to replace it within a few months.
Is there any way to migrate my account to the new PC when I want to move? or do I need to buy license one by one?

### Addon & CoApp version and details

question

**Comment by jcfrog** [comment]
Hi,
You don't need to migrate your license, you can use it on all your devices. Just revalidate your license key on the new PC: https://github.com/aclap-dev/video-downloadhelper/wiki/V9-interface#validate-your-license-key
